FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -17,11 +17,22 @@ const UniversalFeed = () => {
         commentPlaceholder: 'Comment',
         noCommentPlaceholder: 'Add Comment',
         commentCount: 0,
-        footerTextStyle: {}
+        footerTextStyle: {},
+        authorName: 'Theresa Webb',
+        postedTime: '2h',
+        showEdited: true,
+        showLabel: true,
+        labelType: 'Admin',
+        showPin: false,
+        mediaUrl:'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSJhNhTcdpoN6c-rzLj336_o2WpgLgeqirPchSSBerB&s',
+        postText:`Here is a list of social media tools to help you get started with your marketing initiatives.`,
+        attachment_type: [3]
```

Comment:
why hard-coding this values?

Reply:
These are also dummy params, will be removed after api implementation.